app.py

```diff
@@ -130,3 +131,37 @@ def index():
         logger.critical("Caught unhandled exception.", exc_info=True)
 
     return ""
+
+
+@app.schedule("cron(5 0 * * *)")
+def check_user_locks():
+    api_url = "http://localhost:8010/users"
+    res = requests.get(url=api_url)
+    assert res.status_code == 200
+
+    now = datetime.now()
+    if now.month == 1:
+        now.month = 12
+        now.year = now.year - 1
+    else:
+        now.month -= 1
+
+    month = f"{now.year}-{now.month}"
+
+    for user_id, name in res.json().items():
+        lock_url = f"http://localhost:8010/users/{user_id}/locks/{month}"
+        res = requests.get(url=api_url)
+        assert res.status_code == 200
+
+        if len(res) <= 1:
+            headers = {"Content-Type": "application/json"}
+            res = requests.post(
+                url="https://slack.com/api/chat.postMessage",
+                json=dict(
+                    text=f"Not locked {month}",
+                    token=config["bot_access_token"],
+                    channel=user_id,
+                ),
+                headers=headers,
+            )
+            return res.status_code
```

Comment:
This should be `assert res.status_code == 200` (and changed to an actual check/log later)